porting/npl/riot/include/nimble/nimble_npl_os.h

```diff
@@ -100,7 +100,8 @@ ble_npl_eventq_get(struct ble_npl_eventq *evq, ble_npl_time_t tmo)
     } else if (tmo == BLE_NPL_TIME_FOREVER) {
         return (struct ble_npl_event *)event_wait(&evq->q);
     } else {
-        return (struct ble_npl_event *)event_wait_timeout(&evq->q, (tmo * 1000));
+        return (struct ble_npl_event *)event_wait_timeout(&evq->q,
+                                                          (tmo * US_PER_MS));
```

Comment:
If I am not mistaken, a source for error remains for this call: if `tmo`s value is greater than (UINT32T_MAX / 1000 -> 1,19 hours), the timeout will be set incorrectly. Looking at RIOTs µs-based timer APIs it seems this can not be so easily solved. But how about we a) put in an assert that triggers once this overflow happens, or b) return NULL in that case, which subsequently would also in most cases trigger an assertion by the calling code...

Reply:
I have to admit that I intentionally ignored the edge case here, as you mention, there is no easy way to solve the edge case here.

Reply:
I'm not that familiar with Bluetooth timings, do you consider it likely that this assertion will trigger during the lifetime of a node? Are there any timing durations in the Bluetooth spec itself that are longer than this 1.19 hours?